### PR TITLE
Application Profond

### PR DESCRIPTION
**Pull request by emarai**
#### Project Abstract

[Profond.ai](http://profond.ai/) is web3 tools to help developers and creators to build, validate, and scale their dApps. We provide smart contract builder (NFT and token), analytic, and indexer. We will provide: 

1. Create smart contracts with no-code interface for Fungible Token and NFT
2. Analytic for smart contracts
3. Indexer (coming soon)
4. API Analytics.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [x] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (bank details via email _or_ BTC, Ethereum (USDC/DAI) or Polkadot/Kusama (USDT) address in the application).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

File: applications/Profond.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+### Milestone 2 Example — Analytics and API
+
+- **Estimated Duration:** 1 month
+- **FTE:** 2
+- **Costs:** 6,000 USD
+
+| Number | Deliverable | Specification |
+| --- | --- | --- |
+| 0. | API Documentation | We will provide documentation page on how to create basic NFT collection (PSP34) using our no-code builder and creating a fungible token (PSP22). |
+| 1. | Analytic Functionality | Add on-chain analytic for the smart contract. |
```
Comment (keeganquigley) [inline applications/Profond.md:138]: What exactly does "on-chain analytic" mean? Can you elaborate on this?

File: applications/Profond.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+| 2.  | REST API Service | API for users to get the data analytic. |
+| 3. | Indexer Service | Create an Indexer service. |
+| 4. | Go To Market | Onboard 20 project within a month after release. |
```
Comment (keeganquigley) [inline applications/Profond.md:141]: I'm not sure this is something that is covered by the grants program, you can probably remove the "go to market" part.

File: applications/Profond.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+- **Costs:** 4,000 USD
+
+| Number | Deliverable | Specification |
+| --- | --- | --- |
+| 0a. | License | MIT |
+| 0b. | Documentation | We will provide documentation page on how to create NFT (PSP34) and FT (PSP22) smart contract with our interface. API documentation for project settings will also be provided.  |
+| 0c. | Testing and Testing Guide | All functions will be covered with full integration test, testing guide will also be provided. |
+| 0d. | Article | We will publish an article that explains the no code builder |
+| 1. | No Code NFT Creation (ink!) | Allows users to create NFT collection smart contract (PSP34) using ink! in one of WASM supported Parachains. |
+| 2. | No Code Fungible Token Creation (ink!) | Allows users to create Fungible Token smart contract (PSP22) using ink! in one of WASM supported Parachains. |
```
Comment (keeganquigley) [inline applications/Profond.md:125]: Cool! Can you expand the technical details a bit in the deliverables? What technologies will you use to build this?

**Comment by keeganquigley** [comment]
Thanks for the application @emarai please see my comments above to expand out the deliverables table. Also one additional question: will you integrate with any Polkadot wallets to sign in or is it strictly by email for now?

File: applications/Profond.md (hunk 1 above)
Comment (ekkirinaldi) [reply applications/Profond.md:138]: It is data analytics for each contract in the project. Users can create a new contract on Profond or put in existing contracts.
Each contract created in Profond will come with analytics. Here's an example of on-chain data analytic for marketplace.paras.near. We're using Metabase for now. We will put this on the proposal.

CleanShot 2023-04-21 at 13 27 18@2x
CleanShot 2023-04-21 at 13 27 59@2x

**Comment by emarai** [comment]
> Thanks for the application @emarai please see my comments above to expand out the deliverables table. Also one additional question: will you integrate with any Polkadot wallets to sign in or is it strictly by email for now?

Yes, we will integrate Polkadot wallets (SubWallet, polkadot.js, Talisman) to sign in.

File: applications/Profond.md (hunk 2 above)
Comment (emarai) [reply applications/Profond.md]: Sure, removed this row.

File: applications/Profond.md (hunk 3 above)
Comment (emarai) [reply applications/Profond.md]: Yep, I have updated the technical details inside the `Specification`.

**Comment by keeganquigley** [comment]
Thanks for the changes @emarai the deliverables look better now. I'll go ahead and mark this application as ready for review and ping the rest of the committee. In the meantime, one more question:

Can you further define what the "indexer service" would look like in M2? Is there a reason you can't use an already existing solution for this?

**Comment by emarai** [comment]
> Thanks for the changes @emarai the deliverables look better now. I'll go ahead and mark this application as ready for review and ping the rest of the committee. In the meantime, one more question:
> 
> Can you further define what the "indexer service" would look like in M2? Is there a reason you can't use an already existing solution for this?

Hey we were thinking of bringing what we know about indexer to polkadot. But, we have found this has been solved by subsquid, so we are removing it from the deliverables.

**Comment by keeganquigley** [comment]
Thanks for the changes @emarai I'm happy to go forward with it.

**Review by keeganquigley — DISMISSED** [review]

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the application, and sorry for the delay here. I'm generally happy to go ahead with it, but could you add the default deliverables (license, tests, documentation etc.) also to the second milestone? Also, could you remove [Sui](https://sui.io/) from the application since we only support solutions for the Polkadot ecosystem?

**Comment by emarai** [comment]
> Thanks for the application, and sorry for the delay here. I'm generally happy to go ahead with it, but could you add the default deliverables (license, tests, documentation etc.) also to the second milestone? Also, could you remove [Sui](https://sui.io/) from the application since we only support solutions for the Polkadot ecosystem?

Hi, thanks for the feedback. We have adjust the application to your comment, please check.

**Review by Noc2 — APPROVED** [review]
Thanks for the update. I'm happy to go ahead with it.

**Review by randombishop — APPROVED** [review]

**Comment by keeganquigley** [comment]
Hi @emarai how is milestone 1 coming along?

**Comment by emarai** [comment]
Hi @keeganquigley we are still working on it, it is taking us more time than our previous estimation. Our target would be to submit the result at the end of this month.

**Pull request merged**